app/celery/service_callback_tasks.py

```diff
@@ -228,14 +224,14 @@ def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
             )
             raise e
     except NonRetryableException as e:
-        current_app.logger.error(
+        current_app.logger.critical(
             "Not retrying: %s failed for %s, url: %s. exc: %s",
             self.name,
             logging_tags,
             service_callback.url,
             e,
         )
-        raise e
+        raise
```

Comment:
Ditto: Why raise for some NonRetryableExceptions but not others?

Reply:
I deleted all of them then hit issues and put them back. Changing the retry functionality was out of scope anyways, I just wanted to fix logging.